extensions/github/src/pushErrorHandler.ts

```diff
@@ -3,7 +3,8 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { commands, env, ProgressLocation, Uri, window } from 'vscode';
+import { TextDecoder } from 'util';
```

Comment:
Isn't `TextDecoder` available in the global scope?

Reply:
I don't think so because the language server marks it as unknown. Also, there's another example [here][ex] in the `git` built-in extension.

[ex]: https://github.com/babakks/vscode/blob/cfad2543487c4a8e8f53b4451dbccdc1c2036f41/extensions/git/src/emoji.ts#L9